_install-and-configure/configuring-opensearch/cluster-settings.md

```diff
@@ -96,6 +96,8 @@ OpenSearch supports the following cluster-level routing and shard allocation set
     - `REPLICA_FIRST` – Replica shards are relocated first, before primary shards. This prioritization may help prevent a cluster's health status from going red when carrying out shard relocation in a mixed-version, segment-replication-enabled OpenSearch cluster. In this situation, primary shards relocated to OpenSearch nodes of a newer version could try to copy segment files to replica shards on an older version of OpenSearch, which would result in shard failure. Relocating replica shards first may help to avoid this in multi-version clusters. 
     - `NO_PREFERENCE` – The default behavior in which the order of shard relocation has no importance. 
 
+- `cluster.allocator.gateway.batch_size` (Integer): Limits the number of shards sent in one batch to data nodes for fetching metadata. Default is `2000`.
```

Comment:
can we change it to -> Limits the number of shards sent in one batch to data nodes for **fetching unassigned shard** metadata